linux/network_device_ops/hot_remove_network_adapter.yml

```diff
@@ -7,6 +7,39 @@
   include_tasks: ../../common/vm_remove_network_adapter.yml
   vars:
     netadapter_mac_addr: "{{ new_network_adapter_mac_addr }}"
+    vm_remove_adapter_ignore_errors: True
+
+- name: "Network adapter hot-remove succeeded"
+  ansible.builtin.debug:
+    msg: "Successfully hot removed new {{ adapter_type }} network adapter {{ new_network_adapter }}"
+  when:
+    - remove_adapter.changed is defined
+    - remove_adapter.changed
+
+- name: "Network adapter hot-remove failed"
+  when: >-
+    (remove_adapter.changed is undefined or
+     not remove_adapter.changed)
+  block:
+    - name: "Known issue - PVRDMA network adapter hot-remove caused kernel crash on Oracle Linux 8.8"
+      ansible.builtin.debug:
+        msg:
+          - "Hot removing PVRDMA network adapter could cause kernel crash on Oracle Linux 8.8."
+          - "Please refer to https://kb.vmware.com/s/article/93131 for details."
+      tags:
+        - known_issue
+      when:
+        - remove_adapter.rc is defined
+        - remove_adapter.rc == 1
```

Comment:
Is it better to use !=0 ?

Reply:
Return code rc != 1 might represent other errors, which is not this known issue.